Sources/SwiftDocC/Model/Rendering/Content/RenderBlockContent.swift

```diff
@@ -250,10 +250,13 @@ public enum RenderBlockContent: Equatable {
     public struct ListItem: Codable, Equatable {
         /// The item content.
         public var content: [RenderBlockContent]
+        /// If this list item is a task list item, whether the task should be checked off.
+        public var checked: Bool?
```

Comment:
Another option would be to re-export the `Checkbox` type directly from Swift-Markdown, which would give us possible values of `.checked`, `.unchecked`, and `nil`. I went for this since this is the Render JSON format i wanted in the end, and this type is directly de-/serialized into Render JSON.